pkg/adaptor/elasticsearch/clients/registry.go

```diff
@@ -0,0 +1,36 @@
+package clients
```

Comment:
issue created to track that work, https://github.com/compose/transporter/issues/234